lua/nvim-treesitter/install.lua

```diff
@@ -302,13 +304,17 @@ local function run_install(cache_folder, install_folder, lang, repo, with_sync,
         },
       })
     end
+    local ts_args = { "generate" }
+    if vim.treesitter.language_version >= 14 then
```

Comment:
I think this might also get important for older nvim as soon as parsers are generated with new ABI

Reply:
What do you mean ?

Reply:
when there are ABI 14 generated parsers, an 13 Neovim could generate the old ABI. But I forgot that 13 is the default so it will do the right thing for now.